Sources/ComposableArchitecture/Internal/TypeName.swift

```diff
@@ -1,15 +0,0 @@
-@usableFromInline
-func typeName(_ type: Any.Type) -> String {
```

Comment:
We have another helper that does a better job that I was utilizing, so deleted this one.